From: John Harwell <john@brainmap.wustl.edu>
Subject: Fixed compilation error for Qt 5.5.1.  QAction's parent parameter was required in Qt 5.5.1 but made optional in a later version of Qt.

--- a/src/GuiQt/BalsaDatabaseUploadSceneFileDialog.cxx
+++ b/src/GuiQt/BalsaDatabaseUploadSceneFileDialog.cxx
@@ -213,7 +213,7 @@ BalsaDatabaseUploadSceneFileDialog::crea
     /*
      * Show password tool button
      */
-    m_showPasswordAction = new QAction("Show");
+    m_showPasswordAction = new QAction("Show", this);
     m_showPasswordAction->setCheckable(true);
     m_showPasswordAction->setChecked(false);
     QObject::connect(m_showPasswordAction, &QAction::triggered,
