From: John Harwell <john@brainmap.wustl.edu>
Subject: Add conditional compilation of 'QOverload" usage only when Qt 5.7 or later.  When not Qt 5.7, older, more verbose syntax for signal connection using function-based connections is used.  Note that some signals are overloaded and in this case, casting is needed to use the corrected overloaded signal and avoid a compilation error.

--- a/src/GuiQt/AnnotationTextSubstitutionViewController.cxx
+++ b/src/GuiQt/AnnotationTextSubstitutionViewController.cxx
@@ -97,8 +97,13 @@ m_browserWindowIndex(browserWindowIndex)
         
         QSpinBox* sb = new QSpinBox();
         sb->setRange(1, 100);
+#if QT_VERSION >= 0x050700
         QObject::connect(sb, QOverload<int>::of(&QSpinBox::valueChanged),
                          [=] { valueIndexSpinBoxChanged(i); } );
+#else
+        QObject::connect(sb, static_cast<void (QSpinBox::*)(int)>(&QSpinBox::valueChanged),
+                         [=] { valueIndexSpinBoxChanged(i); } );
+#endif
         
         QLabel* fl = new QLabel();
         
