From: Yaroslav Halchenko <debian@onerussian.com>
Subject: BF: return NULL (not false) when promised to return VolumeFile*

--- a/src/Files/ImageFile.cxx
+++ b/src/Files/ImageFile.cxx
@@ -1420,7 +1420,7 @@ ImageFile::convertToVolumeFile(const CON
     if ((pixelNormalVector[2] < tinyValue)
         && (pixelNormalVector[2] > -tinyValue)) {
         errorMessageOut = "Control points need to be in a triangular shape; not a line";
-        return false;
+        return NULL;
     }
     
     if (pixelNormalVector[2] < 0.0) {
